ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -93,7 +93,7 @@ public partial class CommandLineBuilder
         public void AppendFileNamesIfNotNull(string[] fileNames, string delimiter) { }
         protected void AppendFileNameWithQuoting(string fileName) { }
         protected void AppendQuotedTextToBuffer(System.Text.StringBuilder buffer, string unquotedTextToAppend) { }
-        protected void AppendSpaceIfNotEmpty() { }
+        protected virtual void AppendSpaceIfNotEmpty() { }
```

Comment:
Hmm. It looks like this is technically a breaking change according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md

Thoughts on whether that's fatal?